src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +79,15 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            onClick={(event) => createMovie(event)}
+            disabled={
+              (
+                !title
+                || !imgUrl
+                || !imdbUrl
+                || !imdbId
+              )
+            }
```

Comment:
```suggestion
            disabled={isDisabled}
```
Store value in separate variable.